### PR TITLE
Remove Sentry from COE claim

### PR DESCRIPTION
**Pull request by mjknight50**
## Summary

- *This work is behind a feature toggle (flipper): NO

In response to some bad logging, I am removing Sentry.  I also see a bug where even though we log the lack of an EDIPI, we continue on to try to create a claim, which is bad.

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/90411
https://github.com/department-of-veterans-affairs/va.gov-team/issues/91111

### CODE REVIEW

File: app/models/saved_claim/coe_claim.rb [hunk 1]
```diff
@@ -1,32 +1,23 @@
...
 
     response = lgy_service.put_application(payload: prepare_form_data)
-    log_message_to_sentry(
-      "COE claim submitted to LGY: #{guid}",
-      :warn,
-      { attachment_id: guid },
-      { team: 'vfs-ebenefits' }
-    )
+
+    Rails.logger.error("COE claim submitted to LGY: #{guid}")
```
Comment (tblackwe) [inline app/models/saved_claim/coe_claim.rb:19]: this should not be `error` level. 

additionally, could you seperate `guid` out to a logger param?
something like: 
`Rails.logger.info("COE claim submitted to LGY", guid:)`

that enables the logs to be searched more easily inside of datadog.

**Review by tblackwe — CHANGES_REQUESTED** [review]

File: app/models/saved_claim/coe_claim.rb (hunk 1 above)
Comment (mjknight50) [reply app/models/saved_claim/coe_claim.rb]: Good catch.  I feel rusty.

File: app/models/saved_claim/coe_claim.rb [hunk 2]
```diff
@@ -1,32 +1,23 @@
...
     if @edipi.blank?
-      log_message_to_sentry(
-        'COE application cannot be submitted without an edipi!',
-        :error,
-        {},
-        { team: 'vfs-ebenefits' }
-      )
+      Rails.logger.error('COE application cannot be submitted without an edipi!')
+      # return back nil and break out of CoeClaim since we don't have a EDIPI
+      return nil
```
Comment (micahaspyr) [inline app/models/saved_claim/coe_claim.rb:14]: No need to put return nil here as the Rails.logger automagically returns nil.  You can delete the return nil statement

**Review by micahaspyr — CHANGES_REQUESTED** [review]
Remove that return nil on line 14

File: app/models/saved_claim/coe_claim.rb (hunk 2 above)
Comment (tblackwe) [reply app/models/saved_claim/coe_claim.rb:14]: the return nil breaks out of the method, without we will still try to put the application

Comment (tblackwe) [reply app/models/saved_claim/coe_claim.rb:14]: could be an `else` instead of `return`

Comment (micahaspyr) [reply app/models/saved_claim/coe_claim.rb:14]: Bah I didn't see that there was something underneath the return nil.

But yes, if/else would be better in this scenario.

**Review by micahaspyr — CHANGES_REQUESTED** [review]

File: app/models/saved_claim/coe_claim.rb (hunk 2 above)
Comment (tblackwe) [reply app/models/saved_claim/coe_claim.rb:14]: now that you call it out, I agree with you. I dislike pre-emptive returns.

**Review by micahaspyr — DISMISSED** [review]
LGTM!

**Review by micahaspyr — APPROVED** [review]
LGTM with all tests passing

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**